gramps/plugins/graph/gvhourglass.py

```diff
@@ -381,3 +402,13 @@ def add_menu_options(self, menu):
         color_family = ColorOption(_('Families'), '#ffffe0')
         color_family.set_help(_('The color to use to display families.'))
         menu.add_option(category_name, 'colorfamilies', color_family)
+
+        ahnentafelorder = BooleanOption(_("Force Ahnentafel order"), False) # 10826
+        ahnentafelorder.set_help(
+            _("Force Sosa / Sosa-Stradonitz / Ahnentafel order for all ancestors where husbands are always on the left."))
```

Comment:
I suggest changing "husbands" to "fathers"
`_("Force Sosa / Sosa-Stradonitz / Ahnentafel layout order for all ancestors, so that fathers are always on the left branch and mothers are on the right branch."))`

Reply:
job done, ty for idea